### DISCUSSION TITLE
Can I update only a certain tag of an image via API?

### DISCUSSION THREAD

**Discussion by nick-traeger** [containrrr/watchtower#1547] [Q&A]
Hello,
since v1.5.0 I can update certain images using the image query parameter. Is it also possible to define the tag that should be updated?
For example, if I send `/v1/update?image=postgres`, all postgres instances will be updated. However, `/v1/update?image=postgres:15` does not update any instance. (Both with and without URL encoding).

**Comment by JensWinkler91** [comment]
Same question here.

I'd like to know whether i define a specifc tag to update my container.
Could you solve it?
